cmd/deck/runlocal

```diff
@@ -77,6 +77,6 @@ fi
     --static-files-location="$DIR/kodata/static" \
     --template-files-location="$DIR/kodata/template" \
     --spyglass-files-location="$DIR/kodata/lenses" \
-    --config-path "${DIR}/../../../config/prow/config.yaml" \
+    --config-path "${DIR}/../../test/integration/config/prow/deck.yaml" \
```

Comment:
When deck is being developed locally, it assumes the prow config at https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml is in the repo which is no longer true. I copied the prod config and stored it next to the integration config.yaml. I can confirm that running `./cmd/deck/runlocal` runs deck locally on my laptop and it works nicely.